config/routes.rb

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
 
 Gamora::Engine.routes.draw do
+  get "amco", to: "authentication#show", as: :authentication
+  get "amco/callback", to: "callback#show", as: :callback
```

Comment:
why not namespaced the routes just idp/?

Reply:
I tried the path convention we follow in the elixir dependency. Routes will be `/auth/amco` and `/auth/amco/callback` because the engine will be mounted like `mount Gamora::Engine => "/auth"`.

IMO this makes clear that we are using the Amco IdP, just idp is too open to interpretations. What do you think?

Reply:
yeah, here is where a more creative name works better that something generic like IDP on projects, ok... carry on

Reply:
Is this the only place `amco` is used? This is a public repo. I don't think we should have our name all over the place. This looks to be a fairly generic provider/tool that is solving a problem we would like to see solved. But it is public and could be useful to others potentially, so why force them to use our name? I think it should be something less specific to us. Why does this library have to care that we are using our idp solution.  If, in the app that uses this library, we want to mount the "engine" in a way that conveys we are using this to connect to an Amco Idp, then we can choose to do that at the app level. Shouldn't be here in the library.